pydictable/core.py

```diff
@@ -83,8 +83,8 @@ def __validate_dict(self, raw_values: dict):
                 field.validate_dict(attr, value)
             except DataValidationError as e:
                 raise DataValidationError(f'{attr}.{e.path}', e.err)
-            except AssertionError:
-                raise DataValidationError(attr, 'Pre check failed. Invalid value "{}" for field "{}"'.format(value, attr))
+            except AssertionError as e:
+                raise DataValidationError(attr, f'Pre check failed: {str(e)}')
```

Comment:
What if the assertion is just `assert a == v` without any message? I think you should keep "Invalid value "{}" for field "{}" and append `str(e)`